modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -83,18 +85,26 @@ export class AvaxP extends BaseCoin {
     return true;
   }
 
-  verifyAddress(params: VerifyAddressOptions): boolean {
-    const bintools = BinTools.getInstance();
-    const address = bintools.parseAddress(params.address, 'P');
-    return address !== undefined;
-  }
-
   /**
    * Check if address is valid, then make sure it matches the root address.
    *
    * @param {VerifyAddressOptions} params address and rootAddress to verify
```

Comment:
nit: avaxp doesn't use root address, let's rename to unlock address

Reply:
done